cmd/minikube/cmd/node_add.go

```diff
@@ -36,13 +41,46 @@ var (
 	cpNode              bool
 	workerNode          bool
 	deleteNodeOnFailure bool
+	osType              string
+	// windowsVersion      string
+
+	osTypeLong = templates.LongDesc(i18n.T(`
+		This flag should only be used when adding a windows node to a cluster.
+
+		Specify the OS of the node to add in the format 'os=OS_TYPE,version=VERSION'. For example, 'os=windows,version=2022'. 
```

Comment:
Put the example here instead as `$ minikube node add --os='os=windows,version=2022'`

Reply:
Already addressed osTypeLong string message!